app/uk/gov/hmrc/gform/commons/MarkDownUtil.scala

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
What is going on in here?
```
app/uk/gov/hmrc/gform/commons/MarkDownUtil.scala
app/uk/gov/hmrc/gform/commons/MarkdownUtil.scala
```

Reply:
Just matched the class name with the file name. It was complaining in IDE

Reply:
Sorry just pushed with the correct renaming